src/index.tsx

```diff
@@ -14,11 +15,12 @@ export type { DefaultAddress, TokenDefaults } from 'hooks/swap/useSyncTokenDefau
 export type { Theme } from 'theme'
 export { darkTheme, defaultTheme, lightTheme } from 'theme'
 
-export type SwapWidgetProps = SwapProps & WidgetProps
+export type SwapWidgetProps = HeaderProps & SwapProps & WidgetProps
```

Comment:
This should be an interface extending the existing props:

```
export interface SwapWidgetProps extends HeaderProps, SwapProps, WidgetProps {}
```

This is slightly easier on the type system: it reduces time for type checking and it surfaces better types downstream.